StreetTreesFoundationKit/STFKMapSnapshot.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  STFKMapSnapshot.swift
```

Comment:
I'm glad you made this class!  I'll modify what I was working on to use this to create the map snapshot on the tree details view controller.